dpnp/backend/kernels/dpnp_krnl_indexing.cpp

```diff
@@ -32,6 +32,33 @@
 #include "dpnpc_memory_adapter.hpp"
 #include "queue_sycl.hpp"
 
+template <typename _DataType1, typename _DataType2>
+class dpnp_choose_c_kernel;
+
+template <typename _DataType1, typename _DataType2>
+void dpnp_choose_c(
+    void* array1_in, const size_t choices_size, void* choices1, void* result1, const size_t size)
```

Comment:
Usually we order parameters in the following way:
`func(result, input_array1, <input_array1_related_parameters>, input_array2, <input_array2_related_parameters>, <remaining_parameters>)`
So I would propose reorder the parameters as below
```suggestion
    void* result1, void* array1_in, void* choices1, const size_t choices_size, const size_t size)
```